dbschema/models.py

```diff
@@ -1,25 +1,36 @@
-from sqlalchemy import Column, Date, Integer, String, DateTime, ForeignKey
-from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.orm import relationship
-
-Base = declarative_base()
-
-
-class Users(Base):
-    __tablename__ = 'users'
-    nicedayuid = Column(Integer, primary_key=True)
-    closed_user_answers = relationship('ClosedUserAnswers')
-    firstname = Column(String)
-    lastname = Column(String)
-    location = Column(String)
-    gender = Column(String)
-    dob = Column(Date)
-
-
-class ClosedUserAnswers(Base):
-    __tablename__ = 'closed_user_answers'
-    id = Column(Integer, primary_key=True)
-    users_nicedayuid = Column(Integer, ForeignKey('users.nicedayuid'))
-    value = Column(Integer)
-    question = Column(String)
-    datetime = Column(DateTime)
+from sqlalchemy import Column, Date, Integer, String, DateTime, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+
+class Users(Base):
+    __tablename__ = 'users'
+    nicedayuid = Column(Integer, primary_key=True)
+    closed_user_answers = relationship('ClosedUserAnswers')
+    firstname = Column(String)
+    lastname = Column(String)
+    location = Column(String)
+    gender = Column(String)
+    dob = Column(Date)
+    
+    user_intervention_state = relationship("UserInterventionState")
+
+
+class ClosedUserAnswers(Base):
+    __tablename__ = 'closed_user_answers'
+    id = Column(Integer, primary_key=True)
+    users_nicedayuid = Column(Integer, ForeignKey('users.nicedayuid'))
+    value = Column(Integer)
+    question = Column(String)
+    datetime = Column(DateTime)
+    
+
+class UserInterventionState(Base):
+    __tablename__ = 'user_intervention_state'
+    id = Column(Integer, primary_key=True, autoincrement=True)

```

Comment:
There is some improvements you can make here because this is a one-to-one mapping (right? a user will ever only have one intervention state), see: https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#one-to-one .